Add migration guide for Email Activations (#67)
WoltLab/woltlab.github.io: pages/migration/wsc-52/migration_wsc-52_php.md

It is strongly recommended to implement `isContentAuthor` within your custom comment manager.
An example implementation [can be found in `ArticleCommentManager`](https://github.com/WoltLab/WCF/blob/aa96d34130d58c150a35ebd8936f09c830ccd685/wcfsetup/install/files/lib/system/comment/manager/ArticleCommentManager.class.php#L213-L219).

## Email Activation

Starting with WoltLab Suite 5.3 the user activation status is independent of the email activation status. 

> A user can be activated even though their mail is not confirmed (and therefore emails are not sent to the user). 

Reviewer: ```suggestion
A user can be activated even though their email address has not been confirmed, preventing emails being sent to these users. 
```
